autohooks/precommit/run.py

```diff
@@ -129,7 +132,7 @@ class ReportProgress:
 
     def __init__(self, progress: Progress, task_id: int) -> None:
         self._progress = progress
-        self._task_id = task_id
+        self._task_id = TaskID(task_id)
```

Comment:
Since the task_id needs to be of the NewType "TaskID" and i am hesitant to touch the type of an parameter, i'm simply converting the int to the newtype TaskID (which is basically also an int)

Reply:
Personally I don't like this kind of typing using [NewType](https://docs.python.org/3/library/typing.html#newtype) but for removing a mypy issue this is fine.